src/app/features/activity-list/components/submitted-transactions-list.layout.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, useMemo } from 'react';
+import { color, Stack, Text } from '@stacks/ui';
+
+import { SubmittedTransactionsItem } from '@app/features/activity-list/components/submitted-transactions-item';
+import { deserializeTransaction } from '@stacks/transactions';
+
+export const SubmittedTransactionsListItem = ({ rawTx, txId }: { rawTx: string; txId: string }) => {
```

Comment:
```suggestion
type SubmittedTransactionsListItemProps = SubmittedTransaction;
export const SubmittedTransactionsListItem = ({ rawTx, txId }: SubmittedTransactionsListItemProps) => {
```

Reply:
Using type here as, for consistency, renaming the interface for use as a components props